src/config.ts

```diff
@@ -16,6 +16,10 @@ export const LOCALSTACK_ENDPOINT: string | undefined = loadOptionalEnvVar(
   "LOCALSTACK_ENDPOINT",
 );
 
+// Redis
+
+export const REDIS_URL: string = loadRequiredEnvVar("REDIS_URL");
```

Comment:
Just a thought but since we're running inside of ECS would we be able to get the endpoint by using the AWS SDK instead of needing to inject an env var?  The AWS credentials are already provided automatically within the ECS environment to leverage the SDK.

Reply:
Yup, we could make that happen using the SDK.  We'd just need to update the ECS task's IAM role to allow it to describe clusters and then something like this call would let us pull out the primary endpoint and port:
```sh
aws elasticache describe-cache-clusters \
    --cache-cluster-id $PRIMARY_REPLICATION_GROUP_ID \
    --show-cache-node-info
```
Want me to go ahead with this change?  The only catch on my side is that without Localstack pro, it'll be hard to locally test this.

Reply:
Since we still need the cache cluster ID let's just stick with injecting the REDIS_URL.